doc/en/user/source/community/json-ld/configuration.rst

```diff
@@ -0,0 +1,82 @@
+JSON-LD Configuration
+=====================
+ 
+Producing the template file
+---------------------------
+
+JSON-LD template file, operate as a mapping level over the stream of features received by a store, transforming them in the desired output.
+This is an example of a json-ld configuration file ::
+
```

Comment:
please use rest code-block syntax so we can enjoy some syntax highlighting:

```
.. code-block: json
   
   {
   "@codext": {
   ....
  
```

Reply:
still waiting on use of code-block here

Reply:
fixed